installation/noobs.md

```diff
@@ -1,50 +1,20 @@
-
-### Download
-
-Alternatively, NOOBS is available for download on the Raspberry Pi website: [raspberrypi.org/downloads](https://www.raspberrypi.org/downloads/)
-
-#### How to install NOOBS on an SD card
-
-Once you've downloaded the NOOBS zip file, you'll need to copy the contents to a formatted SD card on your computer.
-
-To set up a blank SD card with NOOBS:
-
-- Format an SD card as FAT. See the instructions given below.
-  - Your SD card will need to be at least 16GB for Full Raspberry Pi OS, or at least 8GB for all other installs. 
-- Download and extract the files from the NOOBS zip file.
-- Copy the extracted files onto the SD card that you just formatted, so that these files are at the root directory of the SD card. Please note that in some cases it may extract the files into a folder; if this is the case, then please copy across the files from inside the folder rather than the folder itself.
-- On first boot, the "RECOVERY" FAT partition will be automatically resized to a minimum, and a list of OSes that are available to install will be displayed.
-
-#### How to format an SD card as FAT
-
-**Note:** If you're formatting an SD (or micro SD) card that has a capacity over 32GB (i.e. 64GB and above), then see the separate [SDXC formatting](sdxc_formatting.md) instructions.
-
-##### Windows
-
-If you are a Windows user, we recommend formatting your SD card using the SD Association's Formatting Tool, which can be downloaded from [sdcard.org](https://www.sdcard.org/downloads/formatter_4/). Instructions for using the tool are available on the same site.
-
-##### Mac OS
-
-The [SD Association's Formatting Tool](https://www.sdcard.org/downloads/formatter_4/) is also available for Mac users, although the default OS X Disk Utility is also capable of formatting the entire disk. To do this, select the SD card volume and choose `Erase` with `MS-DOS` format.
-
-##### Linux
-
-For Linux users we recommend `gparted` (or the command line version `parted`). Norman Dunbar has written up [instructions](http://qdosmsq.dunbar-it.co.uk/blog/2013/06/noobs-for-raspberry-pi/) for Linux users.
+SD cards with NOOBS pre-installed are available from many of our distributors and independent retailers, including [Pimoroni](https://shop.pimoroni.com/products/noobs-8gb-sd-card), [Adafruit](https://www.adafruit.com/products/1583), and [Pi Hut](http://thepihut.com/collections/raspberry-pi-sd-cards-and-adapters/products/noobs-preinstalled-sd-card).
 
 ## What's included in NOOBS
 
-The following operating systems are currently included in NOOBS:
+The following operating systems are currently included in NOOBS. Note that only Raspberry Pi OS is supplied on the SD card - all other choices require that you have an internet connection to download the operating system.
```

Comment:
ping @XECDesign to confirm which OS images are included on the preloaded NOOBS SD-cards.

Reply:
I'm not sure there's much point spending too much time on revising this page. WRT this specific comment, it's probably easier if I just delete the list of alternative OSes, as it will no doubt change again in the future.

Reply:
I agree it makes sense to remove the list of alternative OSes and replace with more generic-sounding text.
The only reason I was querying the OSes on preloaded SD cards is that at one point "full NOOBS" included both Raspberry Pi OS and LibreELEC, and I have no idea if that's still the case or not...

Reply:
I've removed the list of OSes, and also updated the latst version info as it was out of date. I'm downloading NOOBS now to check if it has LibreELEC built in.

Reply:
You're correct - LibreELEC is included in NOOBS.